cloudflare/resource_cloudflare_custom_hostname.go

```diff
@@ -41,7 +41,7 @@ func resourceCloudflareCustomHostname() *schema.Resource {
 			},
 			"ssl": {
 				Type:     schema.TypeList,
-				Required: true,
```

Comment:
not too familiar with if this is a breaking change / needs to be modified differently? My feeling is that it's OK since it's going from required -> optional and not the other direction